content/en/core/guides/android/branding.md

```diff
@@ -217,7 +217,7 @@ Because the files generated here are signed with the same key that you are going
 
 ### 5. Release the new flavor
 
-Once you have your pull request approved in the cht-android repo, it's recommended to create an alpha version before merge it to master and do the final release.
+Once you have your pull request approved in the cht-android repo, it's recommended to create an alpha version before merge it to main and do the final release.
```

Comment:
Likewise, cht-android is still on `master`.

Reply:
cht-android may be closer than cht-core, but I'll still undo the changes to avoid any confusion.

Reply:
Changes were rolled back for this file